ApollosProject/apollos-prototype
# Adds "Open Livestream" Capability to the MediaControls

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds the ability for the MediaControls on the `ContentItem` to
- Open a media stream, if a content item is live and it has a media stream.
- Open a webview, if a content item is live and there is a web view url. 
- Opens a content's media, if there is media attached. 

### What design trade-offs/decisions were made?

n/a 

### How do I test this PR?
1. Open the most recent sermon item "Spur One Another" 
2. If you have a livestream running, you should see a "Play" button 
  - Otherwise no button
3. If the play button is visible, tapping it should open the webview
  - You can start a livestream by hardcoding `isLive` to true in `data-connector-church-online/src/dataSource.js` 
4. Open a Content Item that isn't live, but has Media (one of the holy spirit videos, for example) 
  - tapping "play" should open the video. 

### What automated tests did you write?

## TODO:

- [x] Closes #839

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
